test/saml20.tests.js

```diff
@@ -23,6 +24,38 @@ describe('saml 2.0 assertion', function () {
     nameIdentifierFormat: 'urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified'
   };
 
+  it('should reject expired cert embedded in saml assertion', function(done){
+    // Turns out the cert we generate expired in 2013
+    let signedAssertion = saml20.create(options);
+    let publicKey = fs.readFileSync(__dirname + '/test-auth0.cer').toString();
+    let saml_passport = new SamlPassport({cert: publicKey, realm: 'urn:myapp', checkRecipient: false, checkCertExpiration: true});
+    saml_passport.validateSamlAssertion(signedAssertion, function(err, profile) {
+      assert.equal(err.message, "SAML certificate not valid.");
+      done();
+    });
+  });
+
+
+  it('should reject expired cert set in options', function(done){
+    // Turns out the cert we generate expired in 2013
+    let signedAssertion = saml20.create(options);
+    let publicKey = fs.readFileSync(__dirname + '/test-auth0.cer').toString();
+    let saml_passport = new SamlPassport({cert: publicKey, realm: 'urn:myapp', checkRecipient: false, checkCertExpiration: true});
+
+    // Kludgy, but now remove the cert
+    samlAssertion = utils.parseSamlResponse(signedAssertion);
```

Comment:
Global var - can you turn on linting in your editor to ensure all globals are caught?